packages/api/src/useSubscription.ts

```diff
@@ -25,15 +25,17 @@ const useSubscription = <T extends TSocketSubscribableEndpointNames>(name: T) =>
             setSubscribed(true);
 
             try {
-                subscriber.current = _subscribe(name, payload).subscribe(
-                    response => {
-                        setData(response);
-                        setIsLoading(false);
-                    },
-                    response => {
-                        setError(response.error);
-                        setIsLoading(false);
-                    }
+                subscribers.current.push(
+                    _subscribe(name, payload).subscribe(
+                        response => {
+                            setData(response);
+                            setIsLoading(false);
+                        },
+                        response => {
+                            setError(response.error);
+                            setIsLoading(false);
+                        }
+                    )
```

Comment:
This call returns an object with `unsubscribe` callback for a particular subscription